### PR TITLE
Use name 'Baseline Site Selection Tool' in the top level documentation.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-# How to use Metis
+# How to use the Baseline Site Selection Tool
 
-This repository includes [Jupyter notebooks](https://jupyter.org/), a Python package, and [Dockerfiles](https://www.docker.com/) for the Metis simulation tools.
+This repository includes [Jupyter notebooks](https://jupyter.org/), a Python package, and [Dockerfiles](https://www.docker.com/) for the simulation tools.
 
 By default the notebooks run with a small amount of demo data, so you can run them anywhere that you can run Jupyter notebooks.
 
 For use of the notebooks to run simulations on your own, larger scale data, you will need to run them on a machine with more CPUs and memory, such as those available via [Terra](https://app.terra.bio/#workspaces/verily-metis/Metis-toolkit-for-vaccine-trial-planning). Please see the computational requirements listed at the top of each notebook.
```
Comment (kleined) [inline README.md:13]: Can we also change the workspace name? Perhaps in a later pull request?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-Different trial designs will have different additional requirements for the simulations. You can reach the Project Metis team at metis@projectbaseline.com to discuss how those changes can be added to this tool.
+Different trial designs will have different additional requirements for the simulations. You can reach the Baseline Site Selection Tool team at metis@projectbaseline.com to discuss how those changes can be added to this tool.
```
Comment (kleined) [inline README.md:15]: Also consider changing?

**Review by kleined — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (deflaux) [reply README.md:13]: Yes, can do.

File: README.md (hunk 2 above)
Comment (deflaux) [reply README.md:15]: Yes, can do.

**Comment by deflaux** [comment]
Test failures are unrelated and due to https://github.com/verilylifesciences/metis/issues/25

File: README.md (hunk 1 above)
Comment (deflaux) [reply README.md:13]: Filed https://github.com/verilylifesciences/metis/issues/27

Comment (kleined) [reply README.md:13]: The package name is also Metis, is that a harder change to make?

Comment (deflaux) [reply README.md:13]: Not at all - easy to do! We can change that name everywhere! Please feel free to make those changes to the code and notebooks if you like. I can take care of the Terra featured workspace and the email group.

Comment (deflaux) [reply README.md:13]: Also, we can change the name of the GitHub repo too. The redirects work fine if someone uses an old link to GitHub.

**Pull request merged**